CodeEdit/InspectorSidebar/Extensions/String+MD5.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  String+MD5.swift
+//  
+//
+//  Created by Nanashi Li on 2022/04/19.
+//
+
+import Foundation
+import CryptoKit
+
+extension String {
+    var MD5: String {
+        let computed = Insecure.MD5.hash(data: self.data(using: .utf8)!)
```

Comment:
Before converting the String to data, leading and trailing whitespaces should be trimmed and it should be converted to lowercased according to gravatar docs.

Proposed changes:

```swift
public extension String {
  func md5(_ lowercased: Bool = true) -> String {
    var trimmed = self.trimmingCharacters(in: .whitespacesAndNewlines)
    if lowercased { trimmed = trimmed.lowercased() }
    let computed = Insecure.MD5.hash(data: trimmed.data(using: .utf8)!)
    return computed.map { String(format: "%02hhx", $0) }.joined()
  }
}
```